sorts.py

```diff
@@ -68,10 +68,33 @@ def chris_insert_sort(values):
 
 
 def peter_insert_sort(values):
-    # TODO: Test for empty input.
-    # TODO: Build sorted linked list from input using Link class.
-    # TODO: Copy sorted linked list into Python list.
-    pass
+    newlist = []
+    if len(values) <= 1:
+        return values
+    else:
+        head = Link(values[0], None)
```

Comment:
Since I changed Link() to use None for nextval when not specified, you don't need to say None here, just Link(values[0]).